clouddriver-ecs/src/integration/resources/testartifacts/createServerGroup-artifact-EC2-TGMappings-multipleLBsAndContainers-artifactFile.json

```diff
@@ -0,0 +1,67 @@
+{
+  "family": "PLACEHOLDER",
+  "containerDefinitions": [
+    {
+      "name": "application1",
+      "image": "PLACEHOLDER",
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "spinnaker-ecs-demo",
+          "awslogs-region": "us-west-2",
+          "awslogs-stream-prefix": "spinnaker"
+        }
+      },
+      "portMappings": [
+        {
+          "protocol": "tcp",
+          "containerPort": 80
+        }
+      ],
+      "environment": [
+        {
+          "name": "PORT",
+          "value": "80"
+        }
+      ],
+      "cpu": 256,
+      "memoryReservation": 512,
+      "essential": true
+    },
+    {
+      "name": "application2",
+      "image": "PLACEHOLDER",
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "spinnaker-ecs-demo",
+          "awslogs-region": "us-west-2",
+          "awslogs-stream-prefix": "spinnaker"
+        }
+      },
+      "portMappings": [
+        {
+          "protocol": "tcp",
+          "containerPort": 84
+        }
+      ],
```

Comment:
Shouldn't we also be specifying hostPort?

Reply:
+1 `bridge` network mode uses both (and they can be different)

Reply:
I was reading through the document and it says "If you are using containers in a task with the bridge network mode, you can specify a non-reserved host port for your container port mapping, or you can omit the hostPort (or set it to 0) while specifying a containerPort and your container automatically receives a port in the ephemeral port range for your container instance operating system and Docker version."
Hence I was trying that scenario in this case. Do I till have to mention the host port?

Reply:
It would be great to have a test that specifically tests host port being a different value.